### PR TITLE
Update project/team names

### PR DESCRIPTION
**Pull request by tvanepps**
This PR corrects inaccurate project/team names or makes them more specific to the eligible work instead of the name of a funding/hosting org.

EF ARG → Applied Research Group (ARG)
EF Devops → ethPandaOps
EF Geth → Geth
EF Ipsilon → Ipsilon
EF Javascript → EthereumJS
EF Portal → Portal Network (EF)
EF Cryptography → Cryptography (EF)
EF Protocol Support → Protocol Support (EF)
EF Security → Protocol Security (EF)
EF Research → Consensus R&D (EF)
EF Robust Incentives Group (RIG) → Robust Incentives Group (RIG) 
EF Stateless Consensus → Stateless Consensus
Prysmatic → Prysm
Status → Nimbus
Status → Codex DAS
Consensys DDS → Dependable Distributed Systems (DDS)

### CODE REVIEW

**Review by terencechain — APPROVED** [review]

**Review by parithosh — APPROVED** [review]

**Review by acolytec3 — APPROVED** [review]

**Review by skylenet — APPROVED** [review]

**Comment by potuz** [comment]
Seeing ARG here is very confusing :argentina:

**Review by potuz — APPROVED** [review]

**Review by rjl493456442 — APPROVED** [review]

File: docs/09-membership.md [hunk 1]
```diff
@@ -96,19 +96,19 @@ If someone is doing work you feel should be eligible but is not currently listed
  | EF Research | [Kevaundray Wedderburn](https://github.com/kevaundray/) | 1 |
  | EF Research | [Luca Zanolini](https://github.com/luca-zanolini) | 1 |
  | EF Research | [Pop Chunhapanya](https://github.com/ppopth/) | 1 |
```
Comment (hwwhww) [inline docs/09-membership.md:98]: "EF Research" can be changed to "EF Consensus R&D" if we are being audited.

Comment (tvanepps) [reply docs/09-membership.md]: added to the PR! thanks

**Comment by fredriksvantes** [comment]
"EF Security" is also not the official name, the team name is "Protocol Security"

**Comment by cheeky-gorilla** [comment]
We discussed this PR on our weekly ops call, and agreed that there should be a broader discussion about if/how to surface the funding org for PG members. However this PR helps make the naming convention more consistent and precise across all the various teams/projects, so we can merge this as-is and then make future updates as needed!

**Pull request merged**